Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:edge
```

Comment:
Yep that maintainer sounds good. It looks good now save for setting the WORKDIR as `/code`